examples/basic_tutorial_BERT.ipynb

```diff
@@ -0,0 +1,1198 @@
+{
```

Comment:
<p>ca you clarify why you compute the gradients (and of what) in a sentence in a .md cell above this one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-deep/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Added some discussion on this. Also: If you know of a more giotto-like way of getting your hands on these gradients I think it would improve this notebook.</p>